packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -15,7 +15,7 @@
 
   .layout {
     display: flex;
-    font-family: var(--terra-consumer-font-family, 'Roboto', sans-serif);
+    font-family: var(--terra-base-font-family, var(--terra-consumer-font-family, 'Roboto', sans-serif));
```

Comment:
this won't work in legacy browsers... just make it roboto and remove the terra-consumer-font-family altogether.

Reply:
Another option would be remove font-family here and import `baseStyles.js` similar to how we do in terra-core components: https://github.com/cerner/terra-core/blob/master/packages/terra-button/src/Button.jsx#L4

You'll also want to [update your version of terra-base](https://github.com/cerner/terra-consumer/blob/master/packages/terra-consumer-layout/package.json#L25-L42) to `v2.x`

Reply:
Yeah I had suggested that earlier... I'm pretty confident that its already pulled in through other terra components already.

Reply:
So, Mike, would you prefer:

1) font-family: var(--terra-base-font-family, 'Roboto', sans-serif);

OR, just leave it hardcoded as before, like:

2) font-family: 'Roboto', sans-serif;

??

Reply:
The ideal solution is that we dont set font-family here at all. Terra already does it in its base style we simply need set the custom property.